app/viewers/embed/viewer/file.rb

```diff
@@ -17,14 +17,6 @@ def height
         super
       end
 
-      def file_type_icon(mimetype)
-        if Constants::FILE_ICON[mimetype].nil?
-          'sul-i-file-new-1'
-        else
-          Constants::FILE_ICON[mimetype]
-        end
-      end
-
       def file_size_text(file_size)
```

Comment:
Move this method above into the common viewer class so we can use it for the PDF viewer too